build.psm1

```diff
@@ -134,11 +134,12 @@ function Get-EnvironmentInformation
 
     if ($Environment.IsLinux) {
         $LinuxInfo = Get-Content /etc/os-release -Raw | ConvertFrom-StringData
+        $LinuxID = lsb_release -is
```

Comment:
lbs_release does'n always exist.  Please handle those cases.

Reply:
Here is an example of how to detect if a command exists:
https://github.com/PowerShell/PowerShell/blob/2f672538b235db9dfd26cba75064f92e1e903fb4/build.psm1#L127

Reply:
I think something like this could be done:

```powershell
$lsb_release = Get-Command lsb_release -Type Application -ErrorAction Ignore
if ($lsb_release) {
  $LinuxID = & $lsb_release -is
}
else {
  $LinuxID = ""
}
```